parallel-install/pkg/engine/engine.go

```diff
@@ -163,8 +163,8 @@ func worker(ctx context.Context, wg *sync.WaitGroup, jobChan <-chan components.C
 				return
 			}
 			if ok {
-				if installationType == "install" {
-					if err := component.InstallComponent(ctx); err != nil {
+				if installationType == "deploy" {
+					if err := component.DeployComponent(ctx); err != nil {
 						component.Status = components.StatusError
 					} else {
 						component.Status = components.StatusInstalled
```

Comment:
I would really consider adding another status here: StatusUpgraded. This provides feedback to library users what actually happened (install vs upgrade)

Reply:
we can think about it in next PR implementing upgrades